src/webgl/gl-constants.js

```diff
@@ -0,0 +1,870 @@
+// WEBGL BUILT-IN TYPES
```

Comment:
> Did you actually go through the GL header and copied them line by line? wow...

Yep, every character was artfully typed by hand...

Truth be told, Sublime has some great party tricks for editing hundreds of lines at the same time.